frontend/src/pages/Inspection/MetaData.jsx

```diff
@@ -1,92 +1,228 @@
+  margin-left: 0.4rem;
+  padding: 0 1rem;
   color: #434faf;
 `;
 
+const FavouriteButton = styled.button`
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: center;
+  background-color: #ffefc5;
+  border: solid 0.1rem #d5ad4e;
+  padding: 0;
+  border-radius: 2.0rem;
+  cursor: pointer;
+  & > p {
+    font-size: 0.8rem;
+    max-width: 6rem;
+    text-align: center;
+    color: #b47b38;
+    padding: 0 0.5rem 0 0.3rem;
+  }
+  &:hover {
+    background-color: #fff7d8;
+  }
+`;
+const LikeCounter = styled.div`
+  border-left: solid 0.1rem #d5ad4e;
+  padding: 0.3rem 0.7rem 0.3rem 0.4rem;
+  font-size: 0.9rem;
+  color: #b47b38;
+`;
+
+const StarFullStyled = styled(StarFull)`
+  height: 1.2rem;
+  color: #b47b38;
+  padding: 0.3rem;
+`;
+const StarEmptyStyled = styled(StarEmpty)`
+  height: 1.2rem;
+  color: #b47b38;
+  padding: 0.3rem;
+`;
+
+const FeedbackLink = styled(Link)`
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: center;
+  background-color: #cee0ff;
+  border: solid 0.1rem #759ed5;
+  padding: 0;
+  border-radius: 2.0rem;
+  cursor: pointer;
+  margin: 0 0.5rem;
+  & > p {
+    font-size: 0.8rem;
+    text-align: center;
+    color: #6082af;
+    padding: 0 0.5rem 0 0.3rem;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  &:hover {
+    background-color: #deedff;
+  }
+`;
+const FeedbackIcon = styled(Book)`
+  height: 1.2rem;
+  color: #6082af;
+  padding: 0.3rem;
+`;
+
 export const MetaData = (props) => {
   const { data, tags, description } = props;
   const date = '20-09-2019';
+
   const {
-    uuid, municipalityName, formatName, url, metadataTypeName, experiencePostGuid,
+    uuid, municipalityName, formatName, url, metadataTypeName, experiencePostGuid, releaseState,
   } = data;
+  const dispatch = useDispatch();
+
+  const [likes, setLikes] = useState(0);
+  const [isLiked, setLiked] = useState(false);
+
+  useState(() => {
+    const internal = async () => {
+      try {
+        const res = await fetch(`/api/Metadata/${uuid}/likes`);
+        const { likeCount, liked } = await res.json();
+        setLikes(likeCount);
+        setLiked(liked);
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch likes for dataset.'));
+        } else {
+          dispatch(alertActions.error('Failed to fetch likes. Please try again later.'));
+        }
+      }
+    };
+    internal();
+  }, []);
+
+  const handleLike = async () => {
+    try {
+      // eslint-disable-next-line no-irregular-whitespace
+      const userSelector = useSelector((state) => state.user);
+      const { token } = userSelector.user;
+      await fetch(`/api/SOMETHING-FOR-ADDING-LIKE/${uuid}/${token}`);
+    } catch (err) {
+      dispatch(alertActions.error('Something went wrong when adding/removing favourite.'));
+    }
+  };
 
   return (
     <Wrapper>
```

Comment:
this should probably be two components "MetadataContent" and "MetadataToolbar".

Reply:
Working on this one.

Reply:
Resolved in new commit now.